Domain/Models/ProjectModel.cs

```diff
@@ -14,6 +14,5 @@ public class ProjectModel : BaseModel
     public string Description { get; set; } = string.Empty;
 
     [Required] public DateOnly StartDate { get; init; }
-
-    public ICollection<TaskModel> Tasks { get; init; } = new List<TaskModel>();
+    [Required] public ICollection<TaskModel> Tasks { get; init; } = new List<TaskModel>();
```

Comment:
why do we need [Required] attribute here?

Reply:
I suggest removing all attributes and using IEntityTypeConfiguration

Reply:
with FluentValidation, am i right? will it work with IOptions pattern?

Reply:
already understood, gonna refactor now

Reply:
You still have attributes, they shouldn't be there.